contracts/VotingToManageEmissionFunds.sol

```diff
@@ -159,14 +165,16 @@ contract VotingToManageEmissionFunds is VotingTo {
         uint256 _emissionReleaseTime, // unix timestamp
         uint256 _emissionReleaseThreshold, // seconds
         uint256 _distributionThreshold, // seconds
-        address _emissionFunds
+        address _emissionFunds,
+        uint256 _minBallotDuration
     ) public onlyOwner {
         require(!initDisabled());
```

Comment:
Seems we don't need this line here anymore because it is already present in `VotingTo._init` function. The same goes for the line `boolStorage[INIT_DISABLED] = true;` below.